ui-admin/src/study/CreateNewCohortModal.tsx

```diff
@@ -0,0 +1,121 @@
+import { useNavContext } from '../navbar/NavContextProvider'
+import React, { useEffect, useState } from 'react'
+import { Portal, PortalStudy } from '@juniper/ui-core'
+import Modal from 'react-bootstrap/Modal'
+import { Button } from '../components/forms/Button'
+import Select from 'react-select'
+import useReactSingleSelect from '../util/react-select-utils'
+import Api, { ExportData } from '../api/api'
+import { useLoadingEffect } from 'api/api-utils'
+import { Field, QueryBuilder } from 'react-querybuilder'
+import 'react-querybuilder/dist/query-builder.scss'
+import { Store } from 'react-notifications-component'
+import { failureNotification } from '../util/notifications'
+
+/**
+ * Returns a cohort builder modal
+ */
+export default function CreateNewCohortModal({ onDismiss }: {onDismiss: () => void}) {
+  const { portalList } = useNavContext()
+  const initialQuery = { combinator: 'and', rules: [] }
+  const [query, setQuery] = useState(initialQuery)
+  const [cohortName, setCohortName] = useState('')
+  const [selectedPortal, setSelectedPortal] = useState<Portal>()
+  const [selectedStudy, setSelectedStudy] = useState<PortalStudy>()
```

Comment:
selectedStudy and selectedPortal should each default if there's only one study/portal available for that user